packages/addons/service/tvheadend42/source/defaults/config

```diff
@@ -1,13 +1,14 @@
 {
-	"uilevel": 1,
-	"uilevel_nochange": false,
+	"wizard": "hello",
+	"uilevel": 0,
+	"uilevel_nochange": true,
 	"ui_quicktips": true,
 	"cookie_expires": 14,
 	"caclient_ui": true,
 	"epg_compress": true,
 	"prefer_picon": true,
 	"chiconpath": "file:///storage/picons/tvh/%C.png",
 	"chiconscheme": 2,
-	"piconpath": "file:///storage/picons/vdr/",
+	"piconpath": "file:///storage/picons/tvh/",
 	"piconscheme": 1
```

Comment:
This isn't a bug, `chiconpath` are snp picons that are only supported from tvh, and `piconpath` are srp picons that are supported from vdr and tvh. The basic idea is that via an addon snp/srp picons coudl be installed and automatically could be used from tvh/vdr (currently kodi has an bug in the addon system which make this impossible).